conf/app.routes

```diff
@@ -143,3 +143,8 @@ GET        /trustees/:index/is-a-uk-citizen                        controllers.T
 POST       /trustees/:index/is-a-uk-citizen                      controllers.TrusteeAUKCitizenController.onSubmit(mode: Mode = NormalMode, index : Int)
 GET        /trustees/:index/change-is-a-uk-citizen                 controllers.TrusteeAUKCitizenController.onPageLoad(mode: Mode = CheckMode, index : Int)
 POST       /trustees/:index/change-is-a-uk-citizen                controllers.TrusteeAUKCitizenController.onSubmit(mode: Mode = CheckMode, index : Int)
+
+GET         /trustees/:index/trustee-live-in-the-uk                        controllers.TrusteeLiveInTheUKController.onPageLoad(mode: Mode = NormalMode, index : Int)
```

Comment:
Can we change the URLs to be `/trustees/:index/live-in-the-uk`?

Reply:
Done